src/main/java/com/amihaiemil/zold/RtWallet.java

```diff
@@ -70,83 +74,61 @@
 
     @Override
     public String getId() throws IOException {
-        final HttpGet walletId = new HttpGet(
```

Comment:
@ammaratef45 , could you please revert back the irrelevant to the `find` method changes throughout the whole class?

Reply:
@denis-konovalyenko :disappointed: I will get rid of VS Code :laughing: 
Sorry, I will revert all of these changes and resubmit the code.